client/src/pages/home/home.css

```diff
@@ -1,3 +1,28 @@
 .home-page{
     background-color: var(--green-secondary);
 }
+
+.adjust-doctor-list{
```

Comment:
Just doctor-list should suffice. Describing functionality names are better used if you reuse these classes/adjustments somewhere else too.

Reply:
Oh I see